### DISCUSSION TITLE
Is it possible to create new windows in Uno.WinUI and desktop platforms?

### DISCUSSION THREAD

**Discussion by manfromarce** [unoplatform/uno#6547] [Q&A]
I have a simple TabView in my application and would like to create a new window when tabs are dragged out.
I'm not sure how to do this because WinUI APIs are different from UWP and should use Window instead of Views, but probably Views for other platforms. 
Does Uno support this feature for MacOS and WinUI (I've read in a previous discussion that multiple views are not supported for Gtk/Linux)?
Thanks in advance

**Comment by jeromelaban** [comment]
At this point, multiple windows APIs are not yet implemented for any platform. This issue is related to what you're asking https://github.com/unoplatform/uno/issues/6515.
